3scale/porta
# Fix label with access codes path

**What this PR does / why we need it**:

Just update the path of the UI screen for developer portal access codes.

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-11378

**Verification steps** 

Set an access code, open the developer portal in incognito mode and verify that the path is right.

**Special notes for your reviewer**:
